webapp/src/components/home/homeContainer.js

```diff
@@ -50,7 +51,7 @@ export default class HomeContainer extends React.Component {
       }
       });
 
-    fetch(webAppConfig.api.url+'/activity',myInit)
+    fetch(webAppConfig.api.url + '/activity?count=20',myInit)

```

Comment:
Nothing should be scrollable - it's a non-interactive dashboard. The panel can overflow if there's more activities than space on the viewport. The viewport should size according to the size of the browser window.